web/src/components/tasks/RaffleBingoPaginator.test.tsx

```diff
@@ -0,0 +1,140 @@
+import { getTaskStatusUpdatedMessage } from "@/lib/utils/helpers";
+import { convertTaskMd } from "@/lib/utils/markdownReader";
+import { generateTask } from "@/test/factories";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import { useMockRoadmap } from "@/test/mock/mockUseRoadmap";
+import { setupStatefulUserDataContext, WithStatefulUserData } from "@/test/mock/withStatefulUserData";
+import { generateUserData } from "@businessnjgovnavigator/shared/test";
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
+
+jest.mock("next/router", () => ({ useRouter: jest.fn() }));
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("@/lib/data-hooks/useRoadmap", () => ({ useRoadmap: jest.fn() }));
+jest.mock("@/lib/roadmap/buildUserRoadmap", () => ({ buildUserRoadmap: jest.fn() }));
+
+const Config = getMergedConfig();
+
+const fetchRaffleBingoTab = async (filename: string): Promise<Task> => {
+  /*
+    This function is a paired down version of the fetchTaskByFilename function to ensure we're getting the
+    real task in the context of a test environment.
+  */
+  const tabTask = await import(`@businessnjgovnavigator/content/roadmaps/raffle-bingo-steps/${filename}.md`);
+
+  return {
+    ...convertTaskMd(tabTask.default),
+    unlockedBy: [],
+    filename,
+  };
+};
+
+let step1: Task;
+let step2: Task;
+
+describe("<RaffleBingoPaginator />", () => {
+  beforeEach(async () => {
+    jest.resetAllMocks();
+    useMockRoadmap({});
+    useMockRouter({});
+    setupStatefulUserDataContext();
+
+    step1 = await fetchRaffleBingoTab("raffle-license-step-1");
+    step2 = await fetchRaffleBingoTab("raffle-license-step-2");
+  });
+
+  const renderPaginator = (task: Task): void => {
+    render(
+      <WithStatefulUserData initialUserData={generateUserData({})}>
+        <RaffleBingoPaginator task={task} />
+      </WithStatefulUserData>
+    );
+  };
+
+  it("renders first tab content when the task is opened", async () => {
+    const task = generateTask({});
+    renderPaginator(task);
+    await waitFor(() => {
+      expect(screen.getByText("Eligibility Criteria")).toBeInTheDocument();
+    });
+    expect(screen.getByRole("button", { name: step1.callToActionText })).toBeInTheDocument();
+    expect(screen.getByRole("button", { name: Config.taskDefaults.continueButtonText })).toBeInTheDocument();
+    expect(
+      screen.queryByRole("button", { name: Config.taskDefaults.backButtonText })
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole("button", { name: Config.taskDefaults.markAsCompleteButtonText })
+    ).not.toBeInTheDocument();
+  });
+
+  it("displays second step task content when on second step", async () => {
+    const task = generateTask({});
+    renderPaginator(task);
+
+    fireEvent.click(screen.getByTestId("stepper-1"));
+    await waitFor(() => {
+      expect(screen.getByText("Application Requirements")).toBeInTheDocument();
+    });
```

Comment:
```suggestion
      expect(screen.findByRole("Application Requirements")).toBeInTheDocument();
```
[dust] No need to make this change but `findByText` will actually do the same thing `waitFor()` does in a less verbose way.

Reply:
@AnJuHyppolite This would be a quick change and I think a good one to include.